app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -86,8 +86,9 @@ class BookmarksToolbar extends ImmutableComponent {
       }
     }
     if (e.dataTransfer.files.length > 0) {
```

Comment:
files only contains the filename, not the full path. We added the full path for files into items when files are present